src/amplitude-client.js

```diff
@@ -1406,6 +1406,25 @@ if (BUILD_COMPAT_2_0) {
   };
 }
 
+/**
+ * Flush all scheduled events.
+ * @public
+ * @param {Amplitude~eventCallback} callback - (optional) callback to run after events are sent.
+ */
+
+ AmplitudeClient.prototype.flushEvents = function flushEvents(callback) {
+  if (this._unsentCount() === 0) {
+    return false;
+  }
+
+  this.sendEvents(callback);
```

Comment:
I don't think sendEvents accepts a callback - you might need to introduce it there.